sdk/cosmosdb/cosmos/src/request/SharedOptions.ts

```diff
@@ -76,4 +76,9 @@ export interface SharedOptions {
    * to serve the request if regular provisioned RUs/second is exhausted.
    */
   disableRUPerMinuteUsage?: boolean;
+
+  /**
+   * Excludes one or more Azure regions for the operation.
```

Comment:
Add a line here, in the samples, and ReadME that this will only be honoured when `enableEndPointDiscovery` is set to `true`